scripts/qa_crowbarsetup.sh

```diff
@@ -4438,8 +4447,10 @@ puts y.to_yaml" > /root/keystone-test-endpoint-update.yaml
     safely crowbar batch --timeout 1500 build < /root/keystone-test-endpoint-update.yaml
 }
 
-function update_keystone_endpoint
+function test_keystone_toggle_ssl
 {
+    prepare_proposal_for_modification $proposal $proposaltype
```

Comment:
I am unable to understand why this change.
`set_keystone_endpoint` creates a batch export in `/root/keystone-test-endpoint-update.yaml` and uses that for another batch run so the `$pfile` created by this call is not really used

and also I could not find `proposal` and `proposal_type` being set for the `testsetup` step

Reply:
If you look at the mkcloud logs you can see errors that it fails to read thr attributes of the ssl cert names because it uses accessor functions that read the $pfile directly. I believe the rest of the failures are a fallout of that.

Reply:
My tests are still running, will check tonight

Reply:
okay, that makes sense. But in batch step these variables will not defined already, so you will end up having 
 ```11:14:36 ++(qa_crowbarsetup.sh:1977) get_proposal_filename(): echo /root/.default.proposal```

The gating job passed because it wasnt a batch job and so the file `keystone.default.proposal` was already there

Here you might want to make it 
``` prepare_proposal_for_modification keystone default ``` 

since we know it is `keystone` and `default`

Reply:
Correct, that was a copy&paste error